doc/try_new_features_before_release.md

```diff
@@ -1,10 +1,16 @@
 # Try new features before release
 
-This documentation shows how to try new features in commit or PR before release even if the PR is not merged.
+This documentation shows how to try new features in commits or PRs before release even if the PR is not merged.
 
-Azure CLI releases new versions every 3 weeks. Many customers and developers are eager to try and test new features before the formal release. Fortunately, Azure CLI project builds artifacts for every commit and PR and you can download them.
+Azure CLI release schedule can be found at https://github.com/Azure/azure-cli/milestones. Many customers and developers are eager to try and test new features before the formal release. Fortunately, Azure CLI project builds artifacts for every commit and PR and you can download them.
 
-## Tutorial for commit
+## Install edge builds for `dev` branch
+
+The most convenient way is to [install edge builds](https://github.com/Azure/azure-cli#edge-builds) released for every commit on `dev` branch.
+
+If no edge builds are released for your platform, you may directly install the artifacts built by the CI pipeline as described below.
```

Comment:
Installing the edge build is certainly easier than digging out the artifacts from ADO.